README.md

```diff
@@ -1,17 +1,277 @@
+
+ ``` cs
+using ReviewService.Abstractions;
+
+/// <summary>
+/// The custom review prompt settings used for prompt conditions.
+/// </summary>
+public record ReviewSettingsCustom : ReviewSettings
+{
+    /// <summary>
+    /// Gets or sets if the application onboarding has been completed.
+    /// </summary>
+    public bool HasCompletedOnboarding { get; init; }
+}
+```
+
+### Add Tracking for Custom Application Events
+
+To track your custom review settings, you can create extensions for `IReviewService` and be sure to make them generic so they are usable with custom review settings.
+
+``` cs
+using ReviewService;
+
+/// <summary>
+/// Extensions of <see cref="IReviewService{TReviewSettings}"/>.
+/// </summary>
+public static class ReviewServiceExtensions
+{
+    /// <summary>
+    /// Tracks that the application onboarding has been completed.
+    /// </summary>
+    /// <typeparam name="TReviewSettings">The type of the object that we use for tracking.</typeparam>
+    /// <param name="reviewService"><see cref="IReviewService{TReviewSettings}"/>.</param>
+    /// <param name="ct">The cancellation token.</param>
+    /// <returns><see cref="Task"/>.</returns>
+    public static async Task TrackOnboardingCompleted<TReviewSettings>(this IReviewService<TReviewSettings> reviewService, CancellationToken ct)
+        where TReviewSettings : ReviewSettingsCustom
+    {
+        await reviewService.UpdateReviewSettings(ct, reviewSettings =>
+        {
+            return reviewSettings with { HasCompletedOnboarding = reviewSettings.HasCompletedOnboarding };
+        });
+    }
+}
+```
+
+#### Built-in Tracking Data
+
+- [PrimaryActionCompletedCount](https://github.com/nventive/ReviewService/blob/1484f946c60e2bf1cb86b27faa60c148a1e56d45/src/ReviewService.Abstractions/ReviewSettings.cs#L13) : The number of primary actions completed.
+- [SecondaryActionCompletedCount](https://github.com/nventive/ReviewService/blob/1484f946c60e2bf1cb86b27faa60c148a1e56d45/src/ReviewService.Abstractions/ReviewSettings.cs#L18) : The number of secondary actions completed.
+- [ApplicationLaunchCount](https://github.com/nventive/ReviewService/blob/1484f946c60e2bf1cb86b27faa60c148a1e56d45/src/ReviewService.Abstractions/ReviewSettings.cs#L23) : The number of times the application has been launched.
+- [ApplicationFirstLaunched](https://github.com/nventive/ReviewService/blob/1484f946c60e2bf1cb86b27faa60c148a1e56d45/src/ReviewService.Abstractions/ReviewSettings.cs#L28) : When the application first started.
+
+### Configure Conditions
+
+If you want to use our default review conditions, you can use `ReviewConditionsBuilder.Default()` and and pass it to the `ReviewService` constructor, or register it as a transient dependency when using dependency injection. Please note that our review conditions are also generic, so they can be used with custom review settings too.
```

Comment:
double "and"

Reply:
![image](https://github.com/nventive/ReviewService/assets/17461593/681e269f-d064-4e8f-a5b4-c6816517f964)
in case github is not clear enough